part-4/parameters-formulas-tags.md

```diff
@@ -0,0 +1,448 @@
+
+Formulas can be re-used, e.g., multiple parameters within a spatial structure can use the same formula.
+
+{% hint style="warning" %}
+If you modify a formula in the editor of a parameter or any other entity, this change will affect all other entities using this formula!
+{% endhint %}
+
+You can re-use formulas across different building blocks by copying and pasting them. To do so, select a formula in the formula list of a building block, press **Ctrl+C**, then move to the formula list of another building block and press **Ctrl+V**.
+
+Upon creation of a formula, it gets the dimension of the parameter or entity where it is created. For parameters, the formula gets the dimension of the parameter. For molecule start values, the dimension is "Amount". For reaction and transport processes kinetics, the dimension is "Amount per time", and for formulas used in event conditions, the dimension is "Dimensionless". The only way to change the dimension of a formula is to change the dimension of the parameter or entity to which the formula is assigned. Furthermore, a formula can only be used in entities or parameters with the same dimension.
+
+{% hint style="info" %}
+It is a good idea to use a name related to the object where the formula is used (e.g., parameter, reaction, observer) - you may even use identical names here.
+{% endhint %}
+
+To enter or edit a **formula string**, click into the unnamed input box above the **Description** field and then use your keyboard. This formula string will be evaluated by the solver once the simulation is run. It is written as a mathematical term that comprises numeric values, arithmetic operation signs, and names of parameters or their alias names. As long as the formula has errors or is incomplete, a red error sign <img src="../assets/icons/ErrorProvider.svg" data-size="line"> is displayed left of the empty input box. Hovering the mouse over this warning symbol will show you a tool tip on the validity of the equation (e.g., missing references or syntax errors).
+
+### Supported Characters and Functions in a Formula‌
+
+In a (explicit) formula, the following characters may be used:
+
+* numbers can be entered as described for constants, e.g. `2.34`; `1.2E-6`; `-150` (**WARNING**: a comma `,` is interpreted as a decimal point!)
+- `TIME` variable: The simulation time
+* the arithmetic operation signs
+    - Plus `+`, e.g., `3 + 5`
+    - Minus `-`, e.g., `3 - 5` or `-5` (unary minus)
+    - Multiplication `*`, e.g., `3 * 5`
+    - Division `/`, e.g., `3 / 5`
+    - Exponentiation `^`, e.g., `3^5` = 3 to the power of 5
+* round brackets **(** **)**, e.g., `(3 + 5) * 2`
+* the constants **pi** and **e**, representing the mathematical constants $\pi$ and $e$.
```

Comment:
I noticed recently that `$..$` does not work with GitBook (the generated latex looks properly in the markdown file, but not in the GitBook-rendered page). https://github.com/Open-Systems-Pharmacology/developer-docs/issues/68
For GitBook we should always use `$$..$$` (then in markdown the latex expression always appears on a new line, but GitBook renders it as **inline** Latex expression as expected)